src/docs/product/accounts/sso/index.mdx

```diff
@@ -37,17 +33,37 @@ Sessions last for [Django's default session length](https://docs.djangoproject.c
 
 ### Google Business App
 
+<Note>
+
+This feature is available only if your organization is on a Team or Trial plan.
```

Comment:
@imatwawana It's also available on a Business plan (Business Plan has everything a Team Plan does and more)

Reply:
Right, Team, Biz, Ent, and Trial plans. But I thought that wording would sound clunky, so I was gonna stick with just saying Team, with the implication that we mean Team and higher. I don't think it's a good idea to just say Team + Biz and not mention Enterprise because we have an Enterprise plan on our pricing page, but then that gets too long. What about this?

```suggestion
This feature is available only if your organization is on a Team plan or higher, or a Trial plan.
```

Reply:
@imatwawana Ok, I'll leave it up to you. Not sure what you do for other parts of the docs but we should be consistent with other featurees.